iothub/device/samples/readme.md

```diff
@@ -72,6 +72,6 @@ You need to clone the repository or download the sample (the one you want to try
 [m-message-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/main/iot-hub/Samples/module/ModuleSample
 [lnk-setup-iot-hub]: https://aka.ms/howtocreateazureiothub
 [lnk-manage-iot-device]: https://github.com/Azure/azure-iot-device-ecosystem/blob/master/setup_iothub.md#create-new-device-in-the-iot-hub-device-identity-registry
-[d-read-d2c-messages-sample]: https://github.com/Azure/azure-iot-sdk-csharp/tree/main/iothub/device/samples/Getting%20Started/ReadD2cMessages
-[d-simulated-device-sample]: https://github.com/Azure/azure-iot-sdk-csharp/tree/main/iothub/device/samples/Getting%20Started/SimulatedDevice
-[d-simulated-device-with-command-sample]: https://github.com/Azure/azure-iot-sdk-csharp/tree/main/iothub/device/samples/Getting%20Started/SimulatedDeviceWithCommand
\ No newline at end of file
+[d-read-d2c-messages-sample]: https://github.com/Azure/azure-iot-sdk-csharp/tree/main/iothub/device/samples/getting%20started/ReadD2cMessages
```

Comment:
@patilsnr capital G and S in main, but not in preview. Which do we want to keep?

Reply:
The other folders are all lower case, so I'd assume we'd want to keep lower case.

Reply:
Lowercase would be my preference.

Reply:
this should be moved to iothub/servicee/samples as well